go.mod

```diff
@@ -58,10 +58,20 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/component-base v0.25.4 // indirect
+	k8s.io/component-base v0.26.0-rc.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20221110221610-a28e98eb7c70 // indirect
 	k8s.io/utils v0.0.0-20221108210102-8e77b1f39fe2 // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace k8s.io/api => k8s.io/api v0.26.0-rc.0
```

Comment:
I think its better to remove this replace section.

Reply:
+1

Reply:
It looks like the script is relying on the replace statements to do special handling for the kubernetes libraries that don't follow the same naming scheme. @pohly do you know if it is safe to skip this logic if the repo does not import k/k?

https://github.com/kubernetes-csi/csi-release-tools/blob/master/go-get-kubernetes.sh#L125

Reply:
If the repo doesn't use k/k, then it should be safe to drop the replace statements.